_pages/install/integrations/open-payment-framework-integration.md

```diff
@@ -0,0 +1,150 @@
+---
+title: SAP Open Payment Framework Integration
+feature:
+  - name: Open Payment Framework Integration
+    spa_version: [NOT_SPECIFIED_YET]
+    cx_version: 2211
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} [NOT_SPECIFIED_YET] {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The Open Payment Framework provides merchants with a streamlined way to connect their chosen Payment Service Provider using configurations.
+
+For more information, see [Open Payment Framework in a Nutshell](https://help.sap.com/docs/SAP_UPSCALE_COMMERCE/0160c41e0de84b218d05bc1185213d1d/5efc3463b4504d27bb9c4fbbb95a4ccc.html?locale=en-US#open-payment-framework-in-a-nutshell) on the SAP Help Portal.
+
+***
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+***
+
+## Requirements
+
+To integrate SAP Open Payment Framework with Composable Storefront, you must have SAP Commerce Cloud 2211 or newer.
+
```

Comment:
Do we know which version of 2211 is required?

Reply:
I cannot specify the exact version required at this moment. Could we perhaps insert a placeholder here and revisit it closer to the OPF release date?